source/rock/RockVersion.ooc

```diff
@@ -4,9 +4,9 @@ RockVersion: class {
 
     getMajor:    static func -> Int    { 1 }
     getMinor:    static func -> Int    { 0 }
-    getPatch:    static func -> Int    { 21 }
+    getPatch:    static func -> Int    { 22 }
     getRevision: static func -> String { "head" }
-    getCodename: static func -> String { "Det. Freppowicz" }
+    getCodename: static func -> String { "<not yet>" }
```

Comment:
Still haven't decided if I want to keep the captain as my primary target, or maybe shift focus to some code reviewer...